text/0000-movecell.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: move_cell
+- Start Date: 2016-06-15
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Extend `Cell` to work with non-`Copy` types.
+
+# Motivation
+[motivation]: #motivation
+
+It allows safe inner-mutability of non-`Copy` types without the overhead of `RefCell`'s reference counting.
```

Comment:
https://github.com/SimonSapin/kuchiki/blob/23e92d45b7406865e035defdc5f5ca9f1809c406/src/tree.rs#L116 is the best example I could find with a quick search.